sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1979,6 +1979,13 @@ object SQLConf {
       .doc("When true, the ArrayExists will follow the three-valued boolean logic.")
       .booleanConf
       .createWithDefault(true)
+
+  private[spark] val CENTRAL_REPOSITORY =
+    ConfigBuilder("spark.sql.centralRepository")
+      .doc("The default central repository used for downloading Hive jars " +
```

Comment:
Users can provide one or multiple remote maven mirror repositories via this parameter. This is not the default central. Check the impl of processRemoteRepoArg

Reply:
I see, but how does this help the problem, if we remove a pointer to the only (flaky?) repo that has the necessary JARs?

Reply:
Yeah. Many different reasons. For example, the maven central could block a certain IP if we download the same JAR too many times. Google mirror does not block the IP in general. Another sad example is the IP could be blocked by Great Firewall when the guys are running it in China.

Reply:
Thanks for the correcting, fix the description in 49ea1cd.